### ISSUE TITLE
JSON fails to parse tabs (\t)

### ISSUE THREAD

**Issue by jackun** [CastagnaIT/plugin.video.netflix#94]
*I'm submitting a ...*
  - [x] bug report
  - [ ] feature request
  - [ ] support request

### Addon version used

0.14.3

### Description

Title thumbnails alt text has tab character in the description and python's json parser fails to parse it.

### Steps to Reproduce

1. Try to login

**Expected behavior:** [What you expected to happen]

Logs in

**Actual behavior:** [What actually happened]

JSON fails to parse control characters

### Installation

* [x] I'm using this Netflix Repo
* [ ] I'm using other Netflix Repo (Please specify which)
* [ ] I'm using a different source (Please tell which)

### Operating System

* [x] Linux (x86/x64)
* [ ] OSX (x86/x64)
* [ ] Windows (x86/x64)
* [ ] Linux (ARM)
* [ ] Android

Calling [json.loads with strict=False](https://github.com/CastagnaIT/plugin.video.netflix/blob/3c54bed54cac40d8cb2e2f1f1ee95044cabb4604/resources/lib/api/website.py#L168) is a quick workaround.

**Comment by CastagnaIT** [comment]
log with debug

**Comment by jackun** [comment]
[kodi.old.log](https://github.com/CastagnaIT/plugin.video.netflix/files/3244737/kodi.old.log)

**Comment by CastagnaIT** [comment]
well fix soon
